oneview_redfish_toolkit/client_session.py

```diff
@@ -39,13 +41,21 @@ def _get_map_clients():
 
 
 def init_map_clients():
-    globals()['map_clients'] = dict()
+    globals()['map_clients'] = OrderedDict()
 
 
 def _set_new_client_by_token(redfish_token, client_ov_by_ip):
     lock = Lock()
     with lock:
-        globals()['map_clients'][redfish_token] = client_ov_by_ip
+        globals()['map_clients'][redfish_token] = {
+            'client_ov_by_ip': client_ov_by_ip,
+            'session_id': str(uuid.uuid4())
+        }
```

Comment:
Even before your change, maybe we have to create a class to replace this dict, its structure it's becoming complicated to understand: RedfishToken->OneViewIP->(OneviewClient, SessionID).